test/README.md

```diff
@@ -379,3 +379,12 @@ The realpath of the 'tmp' directory.
 * return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
 
 Name of the temp directory used by tests.
+
+### WPT
```

Comment:
Maybe `### WPT (Web Platform Tests)` for people who don't know what it is and don't see the link two lines down?